src/Symfony/Component/Form/FormConfigBuilderInterface.php

```diff
@@ -161,6 +161,17 @@ public function setRequired($required);
     public function setPropertyPath($propertyPath);
 
     /**
+     * Sets the property path that the form should be mapped to.
+     *
+     * @param null|string|PropertyPathInterface $propertyPath
+     *             The property path or null if the path should be set
+     *             automatically based on the form's name.
+     *
+     * @return self The configuration object.
+     */
+    public function setMapTo($propertyPath);
```

Comment:
This needs to be documented in the UPGRADE file. See our BC policy on [adding methods to non-API interfaces](http://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces).